tests/cli/multiple-files.test

```diff
@@ -43,7 +43,7 @@ hledger print -f personal.journal -f business.journal -f alias.journal -f person
 
 
 # 3. files can be of different formats
-hledger print -f personal.journal -f a.timeclock -f b.timedot
+hledger print -f personal.journal -f ../journal/a.timeclock -f ../journal/b.timedot
```

Comment:
They failed with `make test`. This change is required, because these journal are in a different directory. (And the way, they were relative even before the change.)